tutorials/grid_coordinates.py

```diff
@@ -0,0 +1,289 @@
+"""
+.. _grid_coordinates:
+
+Grid coordinates
+====================
+
+Grid coordinates in Verde are used to create points on a regularly spaced grid
+that are then used in the spline method to interpolate between sample data
+points. The grid can be specified either by the number of points in each
+dimension (the *shape*) or by the grid node spacing.
+
+Creating regular grids in Verde is done using the :func:`verde.grid_coordinates`
+function. It creates a set of regularly spaced points in both the west-east and
+south-north directions, i.e. a two-dimensional spatial grid.
```

Comment:
@jessepisel how about this slight rephrasing of the two paragraphs above (they would be merged into a single one):

Creating the coordinates for regular grids in Verde is done using the :func:\`verde.grid_coordinates\` function. It creates a set of regularly spaced points in both the west-east and south-north directions, i.e. a two-dimensional spatial grid. These points are then used by the Verde gridders to interpolate between data points. As such, all \`.grid\` methods (like :meth:\`verde.Spline.grid\`) take as input the configuration parameters for :func:\`verde.grid_coordinates\`. The grid can be specified either by the number of points in each dimension (the *shape*) or by the grid node spacing.

Reply:
I like this change, it integrates the gridders into the tutorial and lets the user know why they are using `grid_coordinates`